core/src/utils/godot_handler.py

```diff
@@ -54,9 +55,13 @@ def launch_godot(self) -> None:
         if not self.manager.is_leader():
             godot_args.append("--headless")
 
+        random_number = random.randint(0, 10000)
+
```

Comment:
I think that the seed should be an argument of the method instead, it could be optional, such that if not provided we generate it like here